src/tfr_description/xacro/model_arm.xacro

```diff
@@ -0,0 +1,148 @@
+      </geometry>
+      <origin xyz="${-lower_arm_height/2} 0 ${lower_arm_length/2}" />
+    </visual>
+    <collision>
+      <geometry>
+        <box size="${lower_arm_height} ${lower_arm_width} ${lower_arm_length}"/>
+      </geometry>
+      <origin xyz="${-lower_arm_height/2} 0 ${lower_arm_length/2}" />
+    </collision>
+  </link>
+  <joint name="lower_arm_joint" type="revolute">
+    <parent link="turntable" />
+    <child link="lower_arm" />
+    <axis xyz="0 1 0" />
+    <origin xyz="${turntable_radius} 0 ${turntable_height}" />
+    <limit effort="1.0" velocity="1.0" lower="${lower_arm_min}"
+          upper="${lower_arm_max}" />
+  </joint>
+  <transmission name="lower_arm_tran">
+    <type>transmission_interface/SimpleTransmission</type>
+    <joint name="lower_to_turn">
+      <hardwareInterface>hardware_interface/EffortJointInterface
+      </hardwareInterface>
+    </joint>
+    <actuator name="lower_motor">
+      <hardwareInterface>hardware_interface/EffortJointInterface
+      </hardwareInterface>
+      <mechanicalReduction>1</mechanicalReduction>
+      <motorTorqueConstant>1</motorTorqueConstant>
+    </actuator>
+  </transmission>
+
+  <link name="upper_arm">
+    <visual>
+      <geometry>
+        <box size="${upper_arm_height} ${upper_arm_width} ${upper_arm_length}"/>
+      </geometry>
+      <origin xyz="${-upper_arm_height/2} 0 ${upper_arm_length/2}" />
+    </visual>
+    <collision>
+      <geometry>
+        <box size="${upper_arm_height} ${upper_arm_width} ${upper_arm_length}"/>
+      </geometry>
+      <origin xyz="${-upper_arm_height/2} 0 ${upper_arm_length/2}" />
+    </collision>
+  </link>
+  <joint name="upper_arm_joint" type="revolute">
+    <parent link="lower_arm" />
+    <child link="upper_arm" />
+    <axis xyz="0 1 0" />
+    <origin xyz="0 0 ${lower_arm_length}" />
+    <limit effort="1.0" velocity="1.0" lower="${upper_arm_min}"
+          upper="${upper_arm_max}" />
+  </joint>
+  <transmission name="upper_arm_tran">
+    <type>transmission_interface/SimpleTransmission</type>
+    <joint name="upper_to_lower">
+      <hardwareInterface>hardware_interface/EffortJointInterface
+      </hardwareInterface>
+    </joint>
+    <actuator name="upper_motor">
+      <hardwareInterface>hardware_interface/EffortJointInterface
+      </hardwareInterface>
+      <mechanicalReduction>1</mechanicalReduction>
+      <motorTorqueConstant>1</motorTorqueConstant>
+    </actuator>
+  </transmission>
+
+  <link name="scoop" >
+    <visual>
+      <geometry>
+        <box size="${scoop_diameter} ${scoop_width} ${scoop_diameter/2}" />
+      </geometry>
+      <origin xyz="${scoop_diameter/2} 0 ${scoop_diameter/4}" />
+    </visual>
+    <collision>
+      <geometry>
+        <box size="${scoop_diameter} ${scoop_width} ${scoop_diameter/2}" />
+      </geometry>
+      <origin xyz="${scoop_diameter/2} 0 ${scoop_diameter/4}" />
+    </collision>
```

Comment:
Are we giving any of these components inertia?

Reply:
doesnt need to be this pull request, but this might screw with move it.

Reply:
Currently I don't have any, but for setting up Gazebo for Moya I'll need to. That's a separate pull request though, yeah. As far as screwing with MoveIt, I guess we'll find out.

Reply:
I did a little bit of that already, there are a few kinks but check out that proto model I checked in.